Dockerfile

```diff
@@ -10,15 +10,38 @@ USER node
 
 RUN yarn install
 
-RUN yarn bootstrap
+RUN yarn workspace @zooniverse/react-components build
 
-FROM node:12 AS production-apps
+RUN yarn workspace @zooniverse/classifier build
```

Comment:
Setting those envs will actually break these builds. That's one of the questions I have about how our build process works. Why do we bootstrap the monorepo in one environment, then deploy the NExt apps in a different environment?
Shouldn't we build and run everything in the same environment. I could see everything failing, for example, if we built for staging/development but ran in a production environment, mostly because the classifier build wouldn't match the runtime environment.